src/main/java/games/strategy/triplea/ui/AbstractUndoableMovesPanel.java

```diff
@@ -72,9 +72,9 @@ private void initLayout() {
     final JPanel items = new JPanel();
     items.setLayout(new BoxLayout(items, BoxLayout.Y_AXIS));
     // we want the newest move at the top
-    m_moves = new ArrayList<>(m_moves);
-    Collections.reverse(m_moves);
-    final Iterator<AbstractUndoableMove> iter = m_moves.iterator();
+    moves = new ArrayList<>(moves);
+    Collections.reverse(moves);
+    final Iterator<AbstractUndoableMove> iter = moves.iterator();
     if (iter.hasNext()) {
```

Comment:
Not the scope of this PR, but `!moves.isEmpty()` should be used instead of this iterator